posthog/temporal/batch_exports/postgres_batch_export.py

```diff
@@ -81,13 +81,23 @@ class PostgresInsertInputs:
 class PostgreSQLClient:
     """PostgreSQL connection client used in batch exports."""
 
-    def __init__(self, user: str, password: str, host: str, port: int, database: str, has_self_signed_cert: bool):
+    def __init__(
+        self,
+        user: str,
+        password: str,
+        host: str,
+        port: int,
+        database: str,
+        has_self_signed_cert: bool,
+        connection_timeout: int = 30,
```

Comment:
Arbitrary, hard coded. Not sure we want to expose this to users. But maybe make it configurable by us later?